cmake/kmeansConfig.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(OpenMP REQUIRED)
+find_dependency(MPI)
+# Include the Targets file to bring in the exported targets
+include(@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/cmake/kmeans/KmeansTargets.cmake)
```

Comment:
This looks sketchy to me, but if I don't include it, I have to add 

`include(${kmeans_DIR}/kmeansTargets.cmake)
`
after find_package(), in the consumer, which is clearly a hack

Reply:
`find_dependecy` are optional and only if the linkage is `PUBLIC/INTERFACE`, which it isn't.

Use `${CMAKE_CURRENT_LIST_DIR}` otherwise it's no longer relocatable

Reply:
So, remove `find_dependency` cals?

Replace `@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@` with `@{CMAKE_CURRENT_LIST_DIR}@`?

Reply:
`${CMAKE_CURRENT_LIST_DIR} ` it **must not** be evaluated at configure time, but at the execution time (of `find_package`)